src/NuGetGallery/ViewModels/DisplayPackageViewModel.cs

```diff
@@ -35,8 +35,10 @@ public class DisplayPackageViewModel : ListPackageItemViewModel
         public bool IsPackageDeprecationEnabled { get; set; }
         public bool IsPackageRenamesEnabled { get; set; }
         public bool IsGitHubUsageEnabled { get; set; }
+        public bool IsPackageDependentsEnabled { get; set; }
         public NuGetPackageGitHubInformation GitHubDependenciesInformation { get; set; }
         public bool HasEmbeddedIcon { get; set; }
+        public PackageDependents packageDependents { get; set; }
```

Comment:
Please uppercase this to signal it is a public pproperty:

```suggestion
        public PackageDependents PackageDependents { get; set; }
```